package.json

```diff
@@ -6,19 +6,18 @@
   "license": "MIT",
   "scripts": {
     "start": "ts-node src/index.ts",
-    "prod": "tsc && node lib/index.js",
-    "happs:build": "ts-node src/scripts/build-happs.ts build",
-    "happs:clean": "ts-node src/scripts/build-happs.ts clean",
-    "build": "tsc",
+    "deps": "ts-node src/scripts/download-deps.ts",
```

Comment:
What do you think of plugging this into `init` task? The existing task could be renamed to `init:conductor` or sth, to have separation.

Reply:
@danekszy I personally don't want to do that because I may be using symlinks to local versions of core DNAs, and I want to be able to reset Envoy state without re-establishing those symlinks

Reply:
Although if we had a task just for doing what `init` is doing now, I guess that's OK

Reply:
Wanna make that change?

Reply:
If it sits with you, I definitely can :)